dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/exporter/ImageFilterExporter.java

```diff
@@ -41,6 +42,11 @@ public BinaryContentExporterData exportContent(File file, final Map<String, Stri
             parameters.put("filter", filters.keySet().toArray(new String[filters.size()]));
             parameters.put("filters", filters.keySet().toArray(new String[filters.size()]));
             
+            // run pdf filter first (if a pdf)
+            if(!filters.isEmpty() && "pdf".equals(UtilMethods.getFileExtension(file.getName())) && !filters.values().contains(PDFImageFilter.class)) {

```

Comment:
I think this should be encapsulated on:

FileUtil.isPDF(file)

Also wondering if the filters as map can not be filtered by the key instead of doing a O(N) search